### PR TITLE
Streamline `udevadm control` arg. handling

### PR DESCRIPTION
**Pull request by dtardon**
This is a preparation for #25523.

### CODE REVIEW

File: src/udev/udevadm-control.c [hunk 1]
```diff
@@ -21,12 +21,25 @@
...
 #include "virt.h"
 
+char **arg_env = NULL;
+usec_t arg_timeout = 60 * USEC_PER_SEC;
+bool arg_ping = false;
+bool arg_reload = false;
+bool arg_exit = false;
+int arg_max_children = -1;
+int arg_log_level = -1;
+int arg_start_exec_queue = -1;
```
Comment (poettering) [inline src/udev/udevadm-control.c]: these should all be static, please

**Comment by poettering** [comment]
this is a nice improvement. i like!

File: src/udev/udevadm-control.c (hunk 1 above)
Comment (dtardon) [reply src/udev/udevadm-control.c]: Oops, fixed.

**Comment by bluca** [comment]
Haven't seen this before @mrc0mmand ?

`18:17:07 TEST-17-UDEV: (timeout)`

**Comment by mrc0mmand** [comment]
> Haven't seen this before @mrc0mmand ?
> 
> `18:17:07 TEST-17-UDEV: (timeout)`

Haven't seen this one before either. I restarted the job, let's see if it happens again.

**Comment by mrc0mmand** [comment]
> > Haven't seen this before @mrc0mmand ?
> > `18:17:07 TEST-17-UDEV: (timeout)`
> 
> Haven't seen this one before either. I restarted the job, let's see if it happens again.

Looks like it was indeed a flake, but I'll keep an eye on it in case it pops up again somewhere else.

**Review by bluca — APPROVED** [review]

**Review by yuwata — COMMENTED** [review]
Belated LGTM. Thank you.

**Comment by YHNdnzj** [comment]
> Looks like it was indeed a flake, but I'll keep an eye on it in case it pops up again somewhere else.

Occurred again in https://jenkins-systemd.apps.ocp.cloud.ci.centos.org/job/upstream-vagrant-archlinux/7512/

**Comment by mrc0mmand** [comment]
> > Looks like it was indeed a flake, but I'll keep an eye on it in case it pops up again somewhere else.
> 
> Occurred again in https://jenkins-systemd.apps.ocp.cloud.ci.centos.org/job/upstream-vagrant-archlinux/7512/

I can actually reproduce this quite reliably in CI, lemme try to do a bisect.

**Pull request merged**